pkg/common/helper/helper.go

```diff
@@ -162,6 +136,36 @@ func (h *H) Setup() error {
 	return nil
 }
 
+// SetupNewProject creates new project prefixed with osde2e-. Also creates essential serviceaccounts and rolebindings in the project.
+func (h *H) SetupNewProject(ctx context.Context, suffix string) (*projectv1.Project, error) {
+	var err error
+	// setup project and dedicated-admin account to run tests
+	// the service account is provisioned but only used when specified
+	var v1project *projectv1.Project
+	ginkgo.GinkgoLogr.Info("Setup called for %s", "osde2e-"+suffix)
+	v1project, err = h.createProject(ctx, suffix)
+	if h.OutsideGinkgo && err != nil {
+		return nil, fmt.Errorf("failed to create project: %s", err.Error())
+	}
+	Expect(err).ShouldNot(HaveOccurred(), "failed to create project")
+	Expect(v1project).ShouldNot(BeNil())
+
+	h.CreateServiceAccountsInProject(ctx, v1project)
+
+	// Quick fix for Hypershift pipelines failing. Currently the RBAC is not being created in the projects.
+	if !viper.GetBool(config.Hypershift) {
+		err = wait.PollImmediate(1*time.Second, 1*time.Minute, func() (bool, error) {
+			_, err = h.Kube().RbacV1().RoleBindings(v1project.Name).Get(ctx, "dedicated-admins-project-dedicated-admins", metav1.GetOptions{})
+			if apierrors.IsNotFound(err) {
+				return false, nil
+			}
+			return true, err
+		})
+		Expect(err).NotTo(HaveOccurred())
```

Comment:
This error check needs to be gated like line 147 to not error check with gomega when we are outside of ginkgo

Reply:
Yeah, let's just go ahead and add the same conditional. It will blow up if we ever run with this specific configuration (hypershift + outside of ginkgo)